pykern/pkunit.py

```diff
@@ -351,7 +351,8 @@ def pkre(expect_re, actual, flags=re.IGNORECASE + re.DOTALL):
         actual (object): run-time value
         flags: passed on to re.search [IGNORECASE + DOTALL]
     """
-    if not re.search(expect_re, actual, flags=flags):
+    from pykern import pkcompat
+    if not re.search(expect_re, pkcompat.from_bytes(str(actual)), flags=flags):
```

Comment:
I don't think this is correct `pkcompat.from_bytes(str(actual))`. from_bytes takes a bytes argument but str returns a string. You can remove the str call, and I think it should give you what you want. pkcompat.from_bytes is robust to the fact that it might be given a string.

Reply:
7918c4a